JS Async Functions: Relay Race Challenge (#106)
neuefische/web-exercises: sessions/js-async-functions/relay-race/README.md

# Relay Race 🏃‍♀️

## Task

Animate the relay runners along the race track using the `animate()` function.

> The first runner can start immediately once the "🏁 Start race" button was clicked.

Reviewer: ```suggestion
The first runner can start immediately after the "🏁 Start race" button was clicked.
```

> The second runner should only start after the they have been reached by the first one (the first animation has finished).

Reviewer: ```suggestion
The second runner should start after the first runner has reached them (in other words: when the first animation has finished).
```

> The same should be true for the third runner in relation to the second one.

Reviewer: ```suggestion
The same goes for the third runner in relation to the second one.
```

Switch to the [index.js](./js/index.js) file and update the code in the click handler!

## Notes

> - You only have to touch the `js/index.js` file.

Reviewer: ```suggestion
- You only need to touch the `js/index.js` file.
```
